Header header
string filename
sensor_msgs/Image image
sensor_msgs/PointCloud2 pcloud
